projects/recipe-page/phase-3-bootstrap-prompt.md

```diff
@@ -94,6 +94,8 @@ Save the file, then refresh `index.html` in Chrome by pressing `<COMMAND> + r`.
 
 Do you remember how to tell Bootstrap to allow a div to take up a certain number of columns? On a small screen, we usually want divs to take up all 12 columns so things are as large as possible. Let's add the `col-xs-12` class to the divs that currently have the `top`, `left` and `right` classes. (Remember, elements may have more than one class!) Here's an abbreviated example:
 
+Don't forget to add your <div class=”row”> before your column <div>
```

Comment:
is this a note, or part of the code? Putting it after the "Here's an example:" is a little confusing -  Maybe add it to the other "remember" statement in the parenthesis just above?

Reply:
Yes I think that suggestion will work! To just add it to the "remember statement"